### PR TITLE
Learn hub page

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

New learn hub page

### CODE REVIEW

File: src/pages/learn/index.tsx [hunk 1]
```diff
@@ -0,0 +1,729 @@
...
+                    <GatsbyImage image={getImage(data.dogeComputer)} />
+                  </Center>
+                  <ButtonLink to="/security">Crypto security 101</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+
+            <Banner>
+              <BannerBody>
+                <h3>Things to consider</h3>
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:359]: @konopkja slight preference for being more explicit here. headings and their content should mostly make sense in isolation from our context.

```suggestion
                <h3>Things to consider when using Ethereum</h3>
```

File: src/pages/learn/index.tsx [hunk 2]
```diff
@@ -0,0 +1,729 @@
...
+  display: flex;
+  flex-flow: column nowrap;
+  gap: 10px;
+
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    margin: 0;
+  }
+`
+
+const AdditionalReadingHeader = styled.h4`
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:138]: This should be a h3. Its parent is a level 2 heading.

File: src/pages/learn/index.tsx [hunk 3]
```diff
@@ -0,0 +1,729 @@
...
+          <InfoTitle>Learn Hub</InfoTitle>
+          <StakingHomeTableOfContents items={tocItems} />
+        </InfoColumn>
+
+        <ContentContainer id="content">
+          <Section>
+            <h2 id={tocItems[0].id}>{tocItems[0].title}</h2>
+            <p>
+              You have probably heard a thing or two about cryptocurrencies,
+              Bitcoin and blockchain, but do you know what those actually are
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:249]:
```suggestion
              Bitcoin , and blockchain, but do you know what those actually are
```

File: src/pages/learn/index.tsx [hunk 4]
```diff
@@ -0,0 +1,729 @@
...
+            </p>
+            <CardGrid>
+              <Card
+                title="What is Ethereum?"
+                description="If you are new, start here and learn the basics of what
+              crypto and Ethereum are and why they matter."
+              >
+                <>
+                  <Center>
+                    <GatsbyImage image={getImage(data.whatIsEth)} />
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:270]: I know `GatsbyImage` defaults to `alt=""` but IMO we should get into the habit of being explicit with blank alts on images. Otherwise, it's a huge amount of a11y debt to pay back if we migrate to another framework.

```suggestion
                    <GatsbyImage image={getImage(data.whatIsEth)} alt="" />
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
This is looking awesome 😀 mostly nitpick/stylistic suggestions

**Comment by minimalsm** [comment]
Screenshot 2022-06-20 at 12 06 49

Image/button are touching on this card

File: src/pages/learn/index.tsx [hunk 5]
```diff
@@ -0,0 +1,729 @@
...
+              <DocLink to="/">What is Web3?</DocLink>
+              <DocLink to="/">Decentralizing Everything</DocLink>
+            </DocsContainer>
+          </Section>
+
+          <Section>
+            <h2 id={tocItems[1].id}>{tocItems[1].title}</h2>
+            <p>
+              It’s actually quite simple once you get the hang of it. You need
+              an application that is commonly called a “wallet”. Its an app that
```
Comment (lukassim) [inline src/pages/learn/index.tsx:317]:
```suggestion
              an application that is commonly called a “wallet”. It's an app that
```

File: src/pages/learn/index.tsx [hunk 6]
```diff
@@ -0,0 +1,729 @@
...
+              helps you store you funds and to authenticate/interact with
```
Comment (lukassim) [inline src/pages/learn/index.tsx:318]:
```suggestion
              helps you store your funds and to authenticate/interact with
```

File: src/pages/learn/index.tsx [hunk 7]
```diff
@@ -0,0 +1,729 @@
...
+                </>
+              </Card>
+            </CardGrid>
+
+            <Banner>
+              <BannerBody>
+                <h3>Things to consider</h3>
+                <ul>
+                  <li>
+                    Each Ethereum transaction requires a fee in form of ETH,
```
Comment (lukassim) [inline src/pages/learn/index.tsx:362]:
```suggestion
                    Each Ethereum transaction requires a fee in the form of ETH,
```

File: src/pages/learn/index.tsx [hunk 8]
```diff
@@ -0,0 +1,729 @@
...
+                    even if you need to move different tokens built on Ethereum
+                    like USDC or DAI.
+                  </li>
+                  <li>
+                    Fees can be high depending on the demand so we reccomend
```
Comment (lukassim) [inline src/pages/learn/index.tsx:367]:
```suggestion
                    Fees can be high depending on the demand so we recommend
```

File: src/pages/learn/index.tsx [hunk 9]
```diff
@@ -0,0 +1,729 @@
...
+                    using <Link to="/layer-2">Layer 2s</Link>.
+                  </li>
+                  <li>
+                    There are options for custodial and non custodial wallets.
```
Comment (lukassim) [inline src/pages/learn/index.tsx:371]:
```suggestion
                    There are options for custodial and non-custodial wallets.
```

File: src/pages/learn/index.tsx [hunk 10]
```diff
@@ -0,0 +1,729 @@
...
+                </>
+              </Card>
+              <CardGradient
+                title="Emerging usecases"
+                description="There are also other prominent industries being disrupted by blockchain:"
+              >
+                <ul>
+                  <li>
+                    <Link to="https://future.com/what-is-decentralized-science-aka-desci/">
+                      Decentralised Science (Desci)
```
Comment (lukassim) [inline src/pages/learn/index.tsx:461]:
```suggestion
                      Decentralized Science (DeSci)
```

File: src/pages/learn/index.tsx [hunk 11]
```diff
@@ -0,0 +1,729 @@
...
+              >
+                <>
+                  <Center>
+                    <GatsbyImage image={getImage(data.developersEthBlocks)} />
+                  </Center>
+                  <ButtonLink>Explore dApps</ButtonLink>
+                </>
+              </Card>
+              <CardGradient
+                title="Emerging usecases"
```
Comment (lukassim) [inline src/pages/learn/index.tsx:455]:
```suggestion
                title="Emerging use cases"
```

File: src/pages/learn/index.tsx [hunk 12]
```diff
@@ -0,0 +1,729 @@
...
+              </Card>
+              <Card
+                title="Decentralised applications"
+                description="Are creating a new digital economy of peer-to-peer services."
+              >
+                <>
+                  <Center>
+                    <GatsbyImage image={getImage(data.developersEthBlocks)} />
+                  </Center>
+                  <ButtonLink>Explore dApps</ButtonLink>
```
[the remark this answers is not in the document]
Comment (lukassim) [reply src/pages/learn/index.tsx:451]: It seems most people are in favor of using 'dapps'.
We still need to make it consistent throughout the website, will raise on the call today and scope out an issue for this.

File: src/pages/learn/index.tsx [hunk 13]
```diff
@@ -0,0 +1,729 @@
...
+            <h2 id={tocItems[3].id}>{tocItems[3].title}</h2>
+            <p>
+              You can help secure Ethereum and earn rewards at the same time on
+              your ETH by staking it. There are several methods of staking
+              depending on your technical knowledge and the amount of ETH.
+            </p>
+            <CardGrid>
+              <Card
+                title="Staking Ethereum"
+                description="Learn about different ways how to earn rewards by staking ETH."
```
Comment (lukassim) [inline src/pages/learn/index.tsx:507]:
```suggestion
                description="Learn about different ways to earn rewards by staking ETH."
```

File: src/pages/learn/index.tsx [hunk 14]
```diff
@@ -0,0 +1,729 @@
...
+                <>
+                  <Center>
+                    <GatsbyImage image={getImage(data.enterprise)} />
+                  </Center>
+                  <ButtonLink>Explore more</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="How can I get involved?"
+                description="There are technical and non-technical ways how to be part of the community."
```
Comment (lukassim) [inline src/pages/learn/index.tsx:601]:
```suggestion
                description="There are technical and non-technical ways to be part of the community."
```

File: src/pages/learn/index.tsx [hunk 15]
```diff
@@ -0,0 +1,729 @@
...
+              >
+                <>
+                  <Center>
+                    <GatsbyImage image={getImage(data.eth)} />
+                  </Center>
+                  <ButtonLink to="/eth">What is ETH?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Where to get ETH?"
```
[the remark this answers is not in the document]
Comment (konopkja) [reply src/pages/learn/index.tsx:288]: okay that makes sense

File: src/pages/learn/index.tsx (hunk 4 above)
Comment (pettinarip) [reply src/pages/learn/index.tsx:269]: Good point. I'll try to add an alt to all the images.

**Review by minimalsm — CHANGES_REQUESTED** [review]
Ooops, forgot to submit this yesterday when I did it.

File: src/pages/learn/index.tsx [hunk 16]
```diff
@@ -0,0 +1,929 @@
...
+    },
+  ]
+
+  const heroContent = {
+    title: "Learn Hub",
+    header: "Learn about Ethereum",
+    subtitle:
+      "Your educational guide to the world of Ethereum. Learn how Ethereum works and how to connect to it. This page includes technical and non-technical articles, guides, and resources.",
+    image: getImage(data.heroImage),
+    alt: "Illustration of a group of people marvelling at an ether (ETH) glyph in awe.",
```
[the remark this answers is not in the document]
Comment (pettinarip) [reply src/pages/learn/index.tsx:225]: Interesting. Curious about why is it like that? is the reader going to tell some generic image description to the user?

[the remark this answers is not in the document]
Comment (minimalsm) [reply src/pages/learn/index.tsx:225]: @pettinarip screen readers will completely ignore empty alts (i.e. alt="") because it treats them as decorative. You can imagine, as a screen reader user, that hearing descriptions of decorative images hundreds of times per day would get pretty annoying 😝

[Here is the W3 guidelines on decorative images.](https://www.w3.org/WAI/tutorials/images/decorative/)

File: src/pages/learn/index.tsx [hunk 17]
```diff
@@ -0,0 +1,943 @@
...
+              You have probably heard a thing or two about cryptocurrencies,
+              Bitcoin, and blockchain, but do you know what those actually are
+              and how they relate to Ethereum? And what is Ethereum anyway?{" "}
+            </p>
+            <p>
+              <Link to="/what-is-ethereum/">
+                Check out our introduction to Ethereum
+              </Link>
+              .
+            </p>
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:263]:
```suggestion
            <p>
              You have probably heard a thing or two about cryptocurrencies,
              Bitcoin, and blockchain, but do you know what those actually are
              and how they relate to Ethereum? <Link to="/what-is-ethereum/">And what is Ethereum anyway?</Link>
            <p>
            </p>
```

File: src/pages/learn/index.tsx [hunk 18]
```diff
@@ -0,0 +1,943 @@
...
+                      image={getImage(data.eth)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/eth/">What is ETH?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Where can I get ETH?"
+                description="There are many ways to aquire ETH depending on your location."
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:300]:
```suggestion
                description="There are many ways to get ETH, depending on your location."
```

File: src/pages/learn/index.tsx [hunk 19]
```diff
@@ -0,0 +1,943 @@
...
+            </CardGrid>
+
+            <AdditionalReadingHeader>
+              Additional reading on Ethereum
+            </AdditionalReadingHeader>
+            <DocsContainer>
+              <DocLink to="/smart-contracts/">
+                What are smart contracts?
+              </DocLink>
+              <DocLink to="/developers/docs//intro-to-ethereum/">
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:321]:
```suggestion
              <DocLink to="/developers/docs/intro-to-ethereum/">
```

File: src/pages/learn/index.tsx [hunk 20]
```diff
@@ -0,0 +1,943 @@
...
+                A technical introduction to Ethereum
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:322]: The general rule I like to follow when ordering a list is ordering roughly from least to the most complex. 

We should put this last on the list IMO.

File: src/pages/learn/index.tsx [hunk 21]
```diff
@@ -0,0 +1,943 @@
...
+              </DocLink>
+              <DocLink to="/web3/">What is Web3?</DocLink>
+              <DocLink
+                to="https://www.youtube.com/watch?v=WSN5BaCzsbo"
+                isExternal
+              >
+                Decentralizing Everything
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:329]: We can bend the rules here and sentence case this for visual consistency.

```suggestion
                Decentralizing everything
```

File: src/pages/learn/index.tsx [hunk 22]
```diff
@@ -0,0 +1,943 @@
...
+                      image={getImage(data.wallet)}
+                      alt="Illustration of a robot."
+                    />
+                  </CardImage>
+                  <ButtonLink to="/wallets/">What is a wallet?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Find a wallet"
+                description="We will help you find the right wallet based on the features you care about."
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:359]: Lighten the tone/fix a preposition at the end of the sentence 

```suggestion
                description="We'll help you find the best wallet based on the features that matter to you."
```

File: src/pages/learn/index.tsx [hunk 23]
```diff
@@ -0,0 +1,943 @@
...
+            <p>
+              It’s actually quite simple once you get the hang of it. You need
+              an application that is commonly called a “wallet”. It's an app
+              that helps you store your funds and to authenticate and interact
+              with applications on Ethereum.
+            </p>
+            <CardGrid>
+              <Card
+                title="What is a wallet?"
+                description="How you manage your ETH and your Ethereum accounts."
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:345]: Awkward phrasing here. I think explaining wallets in terms of accounts here is confusing/a big mental model shift that users struggle with. 

What about this:

```suggestion
                description="Wallets let you manage your ETH and Ethereum applications."
```

File: src/pages/learn/index.tsx [hunk 24]
```diff
@@ -0,0 +1,943 @@
...
+                    Each Ethereum transaction requires a fee in the form of ETH,
+                    even if you need to move different tokens built on Ethereum
+                    like the stablecoins USDC or DAI.
+                  </li>
+                  <li>
+                    Fees can be high depending on the demand so we recommend
+                    using <Link to="/layer-2/">Layer 2s</Link>.
+                  </li>
+                  <li>
+                    There are options for custodial and non-custodial wallets.
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:403]: Can we put an alternative here maybe @konopkja? We're introducing a concept that assumes they've read (and understood) the wallets page. I'd at very least reword this to be jargonless if you feel strongly about keeping the point itself.

File: src/pages/learn/index.tsx [hunk 25]
```diff
@@ -0,0 +1,943 @@
...
+
+            <AdditionalReadingHeader>
+              Additional reading on Ethereum
+            </AdditionalReadingHeader>
+            <DocsContainer>
+              <DocLink to="/community/support/">
+                Ethereum and wallet support
+              </DocLink>
+              <DocLink to="/layer-2/">
+                Layer 2 - reducing transaction cost
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:423]: Consistent terminology (fees/cost)

```suggestion
                Layer 2: reducing transaction fees
```

File: src/pages/learn/index.tsx [hunk 26]
```diff
@@ -0,0 +1,943 @@
...
+                />
+              </BannerImage>
+            </Banner>
+
+            <AdditionalReadingHeader>
+              Additional reading on Ethereum
+            </AdditionalReadingHeader>
+            <DocsContainer>
+              <DocLink to="/community/support/">
+                Ethereum and wallet support
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:420]: Confusing wording:

is it Ethereum AND wallet support (i.e. Ethereum support and wallet support) or Ethereum and wallet support.

```suggestion
                Support for Ethereum and wallets
```

Relevant meme 😄:

image

File: src/pages/learn/index.tsx [hunk 27]
```diff
@@ -0,0 +1,943 @@
...
+          <Section>
+            <h2 id={tocItems[2].id}>{tocItems[2].title}</h2>
+            <p>
+              Ethereum has led to the creation of new products and services that
+              can improve different areas of our lives. We're still in the early
+              stages but there's a lot to be excited about.
+            </p>
+            <CardGrid>
+              <Card
+                title="Decentralized Finance"
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:437]: Any reason we aren't using the acronyms here @konopkja? We should if we want to use the acronym in the button

Screenshot 2022-07-07 at 16 06 28


```suggestion
                title="Decentralized finance (DeFi)"
```

File: src/pages/learn/index.tsx [hunk 28]
```diff
@@ -0,0 +1,943 @@
...
+                description="Explore an alternative financial system that is built without banks and is open to anyone."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.financeTransparent)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/defi/">What is DEFI?</ButtonLink>
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:447]:
```suggestion
                  <ButtonLink to="/defi/">What is DeFi?</ButtonLink>
```

File: src/pages/learn/index.tsx [hunk 29]
```diff
@@ -0,0 +1,943 @@
...
+                description="Are creating a new digital economy of peer-to-peer services."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.developersEthBlocks)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/dapps/">Explore dapps</ButtonLink>
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:505]: Shouldn't this follow the same 'what is ...' pattern as the other cards?

```suggestion
                  <ButtonLink to="/dapps/">What are dapps?</ButtonLink>
```

Screenshot 2022-07-07 at 16 08 51

File: src/pages/learn/index.tsx [hunk 30]
```diff
@@ -0,0 +1,943 @@
...
+                    <GatsbyImage
+                      image={getImage(data.dao)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/dao/">What are DAOs?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Decentralized applications"
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:495]:
```suggestion
                title="Decentralized applications (dapps)"
```

File: src/pages/learn/index.tsx [hunk 31]
```diff
@@ -0,0 +1,943 @@
...
+                description="Are creating a new digital economy of peer-to-peer services."
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:496]:
```suggestion
                description="Dapps are creating a new digital economy of peer-to-peer services."
```

File: src/pages/learn/index.tsx [hunk 32]
```diff
@@ -0,0 +1,943 @@
...
+                      Self-sovereign identity
+                    </Link>
+                  </li>
+                  <li>
+                    <Link to="/social-networks/">
+                      Decentralized social networks
+                    </Link>
+                  </li>
+                  <li>
+                    <Link to="https://future.com/what-is-decentralized-science-aka-desci/">
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:524]: Let's try to get the DeSci page in for this :-)

File: src/pages/learn/index.tsx [hunk 33]
```diff
@@ -0,0 +1,943 @@
...
+                    </Link>
+                  </li>
+                  <li>
+                    <Link to="https://decrypt.co/resources/what-are-play-to-earn-games-how-players-are-making-a-living-with-nfts">
+                      Play-to-earn games
+                    </Link>
+                  </li>
+                  <li>
+                    <Link to="https://woodstockfund.medium.com/quadratic-funding-better-way-to-fund-public-goods-76f1679b2ba2">
+                      Fundraising via Quadratic Funding
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:535]:
```suggestion
                      Fundraising through Quadratic Funding
```

File: src/pages/learn/index.tsx [hunk 34]
```diff
@@ -0,0 +1,943 @@
...
+                    <GatsbyImage
+                      image={getImage(data.infrastructureTransparent)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/nft/">What are NFTs?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Decentralized autonomous organizations"
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:481]:
```suggestion
                title="Decentralized autonomous organizations (DAOs)"
```

File: src/pages/learn/index.tsx [hunk 35]
```diff
@@ -0,0 +1,943 @@
...
+                description="Organizations that run without the need for coordination by a central entity."
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:482]:
```suggestion
                description="DAOs run without the need for coordination by a central entity."
```

File: src/pages/learn/index.tsx [hunk 36]
```diff
@@ -0,0 +1,943 @@
...
+                </>
+              </Card>
+              <CardGradient
+                title="Emerging use cases"
+                description="There are also other prominent industries being disrupted by blockchain:"
+              >
+                <ul>
+                  <li>
+                    <Link to="/decentralized-identity/">
+                      Self-sovereign identity
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:515]:
```suggestion
                      Decentralized identity
```

File: src/pages/learn/index.tsx [hunk 37]
```diff
@@ -0,0 +1,943 @@
...
+                    </Link>
+                  </li>
+                  <li>
+                    <Link to="https://future.com/what-is-decentralized-science-aka-desci/">
+                      Decentralized Science (DeSci)
+                    </Link>
+                  </li>
+                  <li>
+                    <Link to="https://decrypt.co/resources/what-are-play-to-earn-games-how-players-are-making-a-living-with-nfts">
+                      Play-to-earn games
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:530]: Should we introduce the acronym here? No strong opinion from me @konopkja 

```suggestion
                      Play-to-earn games (P2E)
```

File: src/pages/learn/index.tsx [hunk 38]
```diff
@@ -0,0 +1,943 @@
...
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/whitepaper/">Read whitepaper</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+
+            <AdditionalReadingHeader>
+              More about the Ethereum protocol
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:653]: The other headers use the 'Additional reading on...' pattern. We should be consistent. Do we have a preference?

My slight preference is the 'More on...' style as it has a more coherent tone with the writing style/audience we're trying to reach

File: src/pages/learn/index.tsx [hunk 39]
```diff
@@ -0,0 +1,943 @@
...
+                  <Link to="https://github.com/ethereumbook/ethereumbook">
+                    Mastering Ethereum
+                  </Link>{" "}
+                  <em>
+                    December 23, 2018 – Andreas M. Antonopoulos, Gavin Wood
+                    Ph.D.{" "}
+                  </em>
+                </li>
+              </ul>
+              <p>Podcasts covering Ethereum and Cryptocurrencies:</p>
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:790]: This should be a heading

```suggestion
              <h3>Podcasts covering Ethereum and Cryptocurrencies:</h3>
```

File: src/pages/learn/index.tsx [hunk 40]
```diff
@@ -0,0 +1,943 @@
...
+            <h2 id={tocItems[6].id}>{tocItems[6].title}</h2>
+            <div>
+              <p>Books on Ethereum and Cryptocurrencies:</p>
+              <ul>
+                <li>
+                  <Link to="https://www.goodreads.com/book/show/57356067-the-cryptopians">
+                    The Cryptopians: Idealism, Greed, Lies, and the Making of
+                    the First Big Cryptocurrency Craze
+                  </Link>{" "}
+                  <em>February 22, 2022 - Laura Shin</em>
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:736]: We should use `<i>` for italics unless we trying to emphasize the text contained within the tags

```suggestion
                  <i>February 22, 2022 - Laura Shin</i>
```

File: src/pages/learn/index.tsx [hunk 41]
```diff
@@ -0,0 +1,943 @@
...
+                    Million Heist that Almost Destroyed It All
+                  </Link>{" "}
+                  <em>September 29, 2020 - Matthew Leising</em>
+                </li>
+                <li>
+                  <Link to="https://www.goodreads.com/en/book/show/50175330-the-infinite-machine">
+                    The Infinite Machine: How an Army of Crypto-hackers Is
+                    Building the Next Internet with Ethereum
+                  </Link>{" "}
+                  <em>July 14, 2020 - Camila Russo</em>
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:750]:
```suggestion
                  <i>July 14, 2020 - Camila Russo</i>
```

File: src/pages/learn/index.tsx [hunk 42]
```diff
@@ -0,0 +1,943 @@
...
+                    Are Challenging the Global Economic Order
+                  </Link>{" "}
+                  <em>January 12, 2016 - Paul Vigna, Michael J. Casey</em>
+                </li>
+                <li>
+                  <Link to="https://www.goodreads.com/en/book/show/34964890-the-truth-machine">
+                    The Truth Machine: The Blockchain and the Future of
+                    Everything
+                  </Link>{" "}
+                  <em>February 27, 2018 - Paul Vigna, Michael J. Casey</em>
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:764]:
```suggestion
                  <i>February 27, 2018 - Paul Vigna, Michael J. Casey</i>
```

File: src/pages/learn/index.tsx [hunk 43]
```diff
@@ -0,0 +1,943 @@
...
+                </li>
+                <li>
+                  <Link to="https://www.goodreads.com/book/show/23546676-digital-gold">
+                    Digital Gold: Bitcoin and the Inside Story of the Misfits
+                    and Millionaires Trying to Reinvent Money
+                  </Link>{" "}
+                  <em>May 24, 2021 - Nathaniel Popper</em>
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:771]:
```suggestion
                  <i>May 24, 2021 - Nathaniel Popper</i>
```

File: src/pages/learn/index.tsx [hunk 44]
```diff
@@ -0,0 +1,943 @@
...
+
+          <Section>
+            <h2 id={tocItems[6].id}>{tocItems[6].title}</h2>
+            <div>
+              <p>Books on Ethereum and Cryptocurrencies:</p>
+              <ul>
+                <li>
+                  <Link to="https://www.goodreads.com/book/show/57356067-the-cryptopians">
+                    The Cryptopians: Idealism, Greed, Lies, and the Making of
+                    the First Big Cryptocurrency Craze
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:734]: I think we'd be better to remove the subtitles from book titles. I find it very visually overwhelming with the subtitles.

Screenshot 2022-07-07 at 16 17 38

```suggestion
                    The Cryptopians
```

File: src/pages/learn/index.tsx (hunk 27 above)
Comment (konopkja) [reply src/pages/learn/index.tsx:437]: Defi is a shortcut - as such it assumes some prior knowledge, thats who i did not wanted to use the acronym in the title, but thought in the button would be okay.

File: src/pages/learn/index.tsx [hunk 45]
```diff
@@ -0,0 +1,943 @@
...
+              </DocLink>
+            </DocsContainer>
+          </Section>
+
+          <Section>
+            <h2 id={tocItems[5].id}>{tocItems[5].title}</h2>
+            <p>
+              A big portion of Ethereum’s success is thanks to its dedicated
+              community. There are thousands of inspiring and driven people who
+              help push Ethereum’s vision forward. Come and join us!
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:674]:
```suggestion
The success of Ethereum is thanks to its incredibly dedicated community. Thousands of inspiring and driven people help push Ethereum’s vision forward. Come and join us!
```

File: src/pages/learn/index.tsx [hunk 46]
```diff
@@ -0,0 +1,943 @@
...
+                </>
+              </Card>
+            </CardGrid>
+
+            <AdditionalReadingHeader>
+              More about the Ethereum protocol
+            </AdditionalReadingHeader>
+            <DocsContainer>
+              <DocLink to="/developers/">
+                Intro to Ethereum for Developers
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:657]: 1. Lowercase d in developers
2. Is this an intro to Ethereum or intro to developing on Ethereum?
3. Should this be in the protocol section?

```suggestion
                Intro to Ethereum for developers
```

```suggestion
                Ethereum for developers
```

```suggestion
```

File: src/pages/learn/index.tsx (hunk 24 above)
Comment (konopkja) [reply src/pages/learn/index.tsx:403]: @minimalsm  i would keep layer 2s in there, but i think "There are options for custodial and non-custodial wallets." can be removed

File: src/pages/learn/index.tsx [hunk 47]
```diff
@@ -0,0 +1,943 @@
...
+            </CardGrid>
+
+            <Banner>
+              <BannerBody>
+                <h3>Things to consider when using Ethereum</h3>
+                <ul>
+                  <li>
+                    Each Ethereum transaction requires a fee in the form of ETH,
+                    even if you need to move different tokens built on Ethereum
+                    like the stablecoins USDC or DAI.
```
[the remark this answers is not in the document]
Comment (konopkja) [reply src/pages/learn/index.tsx:396]: @minimalsm i disagree here, its not obvious that when you want to send another token, you also have to have ETH. Its in fact quite unintuitive at first.

[the remark this answers is not in the document]
Comment (konopkja) [reply src/pages/learn/index.tsx:396]: please keep original

[the remark this answers is not in the document]
Comment (konopkja) [reply src/pages/learn/index.tsx:396]: i would ask not to merge this change here

File: src/pages/learn/index.tsx (hunk 27 above)
Comment (minimalsm) [reply src/pages/learn/index.tsx:437]: I'm not sure I understand. We're using an acronym (DeFi) in the button without defining what it means. I'm suggesting we add it in parenthesis after the term is fully spelt out. Example: 

Screenshot 2022-07-07 at 17 44 35

File: src/pages/learn/index.tsx [hunk 48]
```diff
@@ -0,0 +1,943 @@
...
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/wallets/find-wallet/">
+                    List of wallets
+                  </ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Crypto security 101"
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:374]:
```suggestion
                title="Crypto security basics"
```

File: src/pages/learn/index.tsx [hunk 49]
```diff
@@ -0,0 +1,943 @@
...
+                description="Learn how to identify scams and how to avoid the most common tricks."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.dogeComputer)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/security/">Crypto security 101</ButtonLink>
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:384]:
```suggestion
                  <ButtonLink to="/security/">Stay secure</ButtonLink>
```

File: src/pages/learn/index.tsx [hunk 50]
```diff
@@ -0,0 +1,943 @@
...
+                Blockchain in developing countries
+              </DocLink>
+            </DocsContainer>
+          </Section>
+
+          <Section>
+            <h2 id={tocItems[3].id}>{tocItems[3].title}</h2>
+            <p>
+              You can help secure Ethereum and earn rewards at the same time on
+              your ETH by staking it. There are several methods of staking
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:564]:
```suggestion
              your ETH by staking it. There are different options for staking
```

File: src/pages/learn/index.tsx [hunk 51]
```diff
@@ -0,0 +1,943 @@
...
+              depending on your technical knowledge and the amount of ETH.
+            </p>
+            <CardGrid>
+              <Card
+                title="Staking Ethereum"
+                description="Learn about different ways to earn rewards by staking ETH."
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:570]:
```suggestion
                description="Learn how to start staking your ETH."
```

File: src/pages/learn/index.tsx [hunk 52]
```diff
@@ -0,0 +1,943 @@
...
+                      image={getImage(data.rhino)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/staking/">Start staking</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Run a node"
+                description="Run a node to play a crucial part in the Ethereum network."
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:584]:
```suggestion
                description="Play a critical role in the Ethereum network by running a node."
```

File: src/pages/learn/index.tsx [hunk 53]
```diff
@@ -0,0 +1,943 @@
...
+                </>
+              </Card>
+            </CardGrid>
+          </Section>
+
+          <Section>
+            <h2 id={tocItems[4].id}>{tocItems[4].title}</h2>
+            <p>
+              For users most interested in the technical part of the Ethereum
+              network.
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:603]: Don't love this but let's rework after the fact

File: src/pages/learn/index.tsx [hunk 54]
```diff
@@ -0,0 +1,943 @@
...
+            </p>
+            <CardGrid>
+              <Card
+                title="Energy Consumption"
+                description="How much energy does it cost to run Ethereum network?"
```
Comment (minimalsm) [inline src/pages/learn/index.tsx:608]:
```suggestion
                description="How much energy does Ethereum use?"
```

**Review by minimalsm — APPROVED** [review]

File: src/pages/learn/index.tsx [hunk 55]
```diff
@@ -0,0 +1,729 @@
...
+                <>
+                  <Center>
+                    <GatsbyImage image={getImage(data.financeTransparent)} />
+                  </Center>
+                  <ButtonLink>What is DEFI?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Stablecoins"
+                description="Stablecoins are cryptocurrencies designed to stay at a fixed value of a dollar."
```
[the remark this answers is not in the document]
Comment (wackerow) [reply src/pages/learn/index.tsx:436]: Yeah.. Agree stablecoins do _not_ need to be pegged to the US dollar. Disagree that they're always pegged to "fiat" too. Fr example, RAI.
